File: Java/Ram.java
class Ram {

    private String capacity;
    private int price;

    Ram(){ }

    Ram(String c, int p){
        this.capacity = c;
        this.price = p;
    }
    
    public String getCapacity() {
        return capacity;
    }
    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    void printDetRam(){
        System.out.println("----------------------");
        System.out.println("Ram Capacity   : " + this.capacity);
        System.out.println("Price          : " + this.price);
    }
}

File: Python/Ram.py
class Ram:

    def __init__(self, capacity, price):
        self.capacity = capacity
        self.price = price

    def setCapacity(self, capacity):
        self.capacity = capacity
    def getCapacity(self):
        return self.capacity

    def setPrice(self, price):
        self.price = price
    def getPrice(self):
        return self.price

    def printDetRam(self):
        print("----------------------")
        print("Ram Capacity   : ", self.capacity)
        print("Price          : ", self.price)